### PR TITLE
Ml

### PR DESCRIPTION
**Pull request by RubenHalman**
@nvuillam this should address the error message and the formatting of the json. Can you verify if the output is now as required for megalinter?

### CODE REVIEW

**Comment by nvuillam** [comment]
@RubenHalman can you post example of output result ? :)

File: README.md [hunk 1]
```diff
@@ -5,76 +5,72 @@ Lightning Flow Scanner (sfdx plug-in)
...
- specify results to ignore. Specify by ruleName and result(if applicable), like shown in the example.
+  --json                                                            set output format as json.
+
+  --loglevel=(trace|debug|info|warn|error|fatal)                    [default: warn] logging level.
 
-#### Example .flowscanignore:
+```
+
+### Configuration file:
+Create a _.flowscanignore_ file to:
```
Comment (nvuillam) [inline README.md:43]: You now have cosmiconfig (from my previous PR), so it won"t be .flowscanignore` but :
- .flow-scanner.yml
- .flow-scanner.json
- etc... ( cf https://github.com/Force-Config-Control/lightning-flow-scanner-sfdx/blob/210100b73947a65c5b0f057a200fd347c459f7ba/src/commands/flow/scan.ts#L145 )

Comment (RubenHalman) [reply README.md:43]: interesting, because when I was testing it worked. Ill update the docs to be sure!

**Comment by nvuillam** [comment]
@RubenHalman if the output is JSON and not human, no need to format, as it is meant for parsing, not reading :)
So all JSON on one line is nice !

File: README.md (hunk 1 above)
Comment (nvuillam) [reply README.md]: @RubenHalman I let .flowscanignore as last option to keep ascending compatibility, but it's better to use more classic  config file names :)

**Comment by RubenHalman** [comment]
I meant to make it readable for you here. How it is shown depends on your cli ofcourse. Does the structure work for you?

**Comment by nvuillam** [comment]
I can copy-paste on https://jsonformatter.curiousconcept.com/# :)

That ok for me, with additional property `severity` it would be perfect :)
Maybe also the full path to the file in a `path` property , to simplify the use with multipackage projects ?

**Pull request merged**